src/bin/gpcheckhdfs/gpcheckhdfs.c

```diff
@@ -6,9 +6,9 @@
  * to you under the Apache License, Version 2.0 (the
  * "License"); you may not use this file except in compliance
  * with the License.  You may obtain a copy of the License at
- * 
+ *
  *   http://www.apache.org/licenses/LICENSE-2.0
- * 
+ *
```

Comment:
just a reminder, we should not update license header unless there is a rat-check failure or necessary changes.

Reply:
thanks!